### ISSUE TITLE
#highlights reload does not work

### ISSUE THREAD

**Issue by HanryuDR** [joemcbride/outlander-osx#153]

**Comment by HanryuDR** [comment]
I edit the highlights file with ATOM, typed #highlights reload but the highlights don't update.

**Comment by joemcbride** [comment]
As in, the highlights you already see on screen don't re-highlight?  Or after they have been reloaded _new_ text does not use the new highlight configuration?

**Comment by HanryuDR** [comment]
New text does not re highlight and if I look at the highlights under prefs, it's still the old ones.

**Comment by joemcbride** [comment]
I just tested this out and it works.  Are you using `#highlight` or `#highlights`?  `#highlight` is the correct command.  You should get a `Highlights reloaded` message on success.

**Comment by joemcbride** [comment]
Looks like the command does catch on `#highlights` but it doesn't process it correctly, which is part of the confusion I'm sure.

**Comment by HanryuDR** [comment]
Yep that was the problem I was using highlightS because that was the name of the file. That works.
What still fails is that the view under prefs does not update (as far as I can tell)

**Comment by joemcbride** [comment]
It appears prefs doesn't properly re-render on its own, but if you click on an updated highlight it should refresh and show the new values.

**Comment by joemcbride** [comment]
I'll add this as a bug to fix the `#highlights` handling and the prefs refresh.

**Comment by joemcbride** [comment]
#110
